joomla/joomla-cms
# [4.2] Missing Folder message

Pull Request for Issue #38866.

### Summary of Changes
Adds a meaningful error message (and alpha-sorts the messages)


### Testing Instructions
Create anew directory in the root of your web space
In the filesystem local plugin add the new directory as a source
Verify that the directory is displayed in the media manager.
Delete the directory from the computer
Reload the media manager

### Actual result BEFORE applying this Pull Request
![image](https://user-images.githubusercontent.com/1296369/193120179-7eb72fd4-031b-4d53-9c6e-cce5932c6ae1.png)



### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/1296369/193120075-55f85f39-0359-467c-83d9-743dd983ef80.png)



### Documentation Changes Required
